### ISSUE TITLE
15 - Create a two column panel view in Asset Browser

### ISSUE THREAD

**Issue by lsemp3d** [o3de/o3de#12855]
**Use Story:**

As a user I need the ability to switch between views. A 1 column view (which we currently have now. All assets in a single column view including folders and assets). And a two column view (left column is just folders of assets. The right column assets and folders from location and downward) - see window explorer for example.

As as user single column views are very hard to parse the details for all folders in a single view. Having a 2 panel view allows user to concentrate on whats most important to them.

For a first past the 2nd column can be detail or thumbnail view. Whichever is free from QT. At the point in which we have a two column view we can start looking at defining further details about whichever view did not come for free

This ticket should also include a button inside the AB to toggle between these two views.

![image](https://user-images.githubusercontent.com/58790905/199326485-3a8fc310-e308-46d6-bec4-4fef6c9f6f3d.png)

**Comment by rainbj** [comment]
See other examples here - https://www.figma.com/file/TfXXZD1xjlw67MMeRYEbi0/Asset-Browser-Design-Document?node-id=862%3A30037

**Comment by MiKom** [comment]
The basic placeholders and interactions for two-column layout got implemented in https://github.com/o3de/o3de/pull/12881 and https://github.com/o3de/o3de/pull/13032
